src/reference/asciidoc/kafka.adoc

```diff
@@ -3174,7 +3174,23 @@ class CustomKafkaConfig implements KafkaListenerConfigurer {
 
     @Override
     public void configureKafkaListeners(KafkaListenerEndpointRegistrar registrar) {
-        registrar.setCustomMethodArgumentResolvers();
+        registrar.setCustomMethodArgumentResolvers(
+        		new HandlerMethodArgumentResolver() {
+
+						@Override
+						public boolean supportsParameter(MethodParameter parameter) {
```

Comment:
Tell, me, please, if you are sure that all of these indents are whitespaces, not tab.

That's a bit confusing , I know: we use tabs in the real code, but whitespaces in docs.
So, 4 spaces for indents in docs please! 😄

Reply:
Now I am sure ;)